src/Psalm/Internal/Analyzer/TypeAnalyzer.php

```diff
@@ -2796,4 +2798,14 @@ public static function simplifyUnionType(Codebase $codebase, Type\Union $union)
 
         return $unique_type;
     }
+
+    /**
+     * @param TLiteralString $atomic_type
+     * @return string
```

Comment:
You can use actual type declarations. Psalm doesn't run on anything below PHP 7.1